docs/COMMUNICATION.rst

```diff
@@ -3,7 +3,7 @@
 Feedback and Communication
 ==========================
 
-Matrix chat room
+Matrix chat room 💬
 ----------------
```

Comment:
```suggestion
Matrix chat room 💬
-------------------
```
📝 ...since the underline needs to be the same length as the title, and apparently emojis count as two spaces.